### PR TITLE
Update the message type URIs from the old did:sov:... format to https://didcomm.org/...

### PR DESCRIPTION
**Pull request by swcurran**
The follow on to the PR that put [RFC 0348](https://github.com/hyperledger/aries-rfcs/tree/master/features/0348-transition-msg-type-to-https) to ADOPTED status, this changes all of the RFCs that contain message types from using the "did:sov..." format to "https://didcomm.org".

The vast majority of the changes are straight text substitutions `s#did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/#https://didcomm.org/#g`. The only RFCs that were more extensively changed are:

- RFC 0005 Protocols, in the section where the Protocol URIs are defined, including details of the `did:sov` resolution mechanism.
- RFC 0020 Message Types, removing the section describing the `did:sov` resolution mechanisms and related references.

This should be a pretty straight forward PR to review, although a little long.

It will likely break some existing open PRs that may need to updated before they can be merged.

### CODE REVIEW

File: concepts/0020-message-types/README.md [hunk 1]
```diff
@@ -29,124 +29,13 @@ Types are specified within a message using the `@type` attribute:
...
-
-The developer would then be able to load this URL in a browser to discover the meaning of `attr_a` and `attr_b`.
+Message types are URIs that may resolve to developer documentation for the message type, as described in [Protocol URIs](../0003-protocols/README.md#message-type-and-protocol-identifier-uris). We recommend that message type URIs be HTTP URLs.
 
 ### Aries Core Message Namespace
 
-> The Aries community is currently in the process of changing the prefix for protocol message types that currently use the did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/ to use https://didcomm.org/. This proces is described in RFC 0348 [Transition Message Type to HTTPs](../../features/0348-transition-msg-type-to-https/README.md).
-
-`did:sov:BzCbsNYhMrjHiqZDTUASHg` will be used to namespace protocols defined by the community as "core protocols" or protocols that agents should minimally support.
+`https://didcomm.org/` is used to namespace protocols defined by the community as "core protocols" or protocols that agents should minimally support.
```
Comment (dhh1128) [inline concepts/0020-message-types/README.md:36]: I don't agree with this summary. There is no such thing as "core protocols." https://didcomm.org is for any protocol that the community develops and maintains, whether it has broad relevance (credential issuance) or narrow relevance (tictactoe, coin flip, etc).

File: concepts/0020-message-types/README.md [hunk 2]
```diff
@@ -29,124 +29,13 @@ Types are specified within a message using the `@type` attribute:
...
 
-This DID is currently held by Daniel Hardman. Ownership will be transferred to the correct entity as soon as possible.
+The `didcomm.org` [DNS entry](https://whois.whoisxmlapi.com/lookup-report/zjRXrYwV5r) is currently controlled by Daniel Hardman.
```
Comment (dhh1128) [inline concepts/0020-message-types/README.md:38]: Control has now been transferred to DIF (about 2 weeks ago).

**Review by dhh1128 — CHANGES_REQUESTED** [review]
Only a couple minor things. Otherwise, this is awesome. Thanks for the hard work that went into it.

File: concepts/0020-message-types/README.md (hunk 1 above)
Comment (swcurran) [reply concepts/0020-message-types/README.md:36]: My feeling is that should be a follow up as it is an editorial change as a separate PR. What is there is a non-editorial transition from the current text.

How do you suggest we handle that?

File: concepts/0020-message-types/README.md (hunk 2 above)
Comment (swcurran) [reply concepts/0020-message-types/README.md:38]: Thanks - I'll update.

File: concepts/0020-message-types/README.md (hunk 1 above)
Comment (dhh1128) [reply concepts/0020-message-types/README.md:36]: Good catch. I agree that I'm raising an issue that should be handled separately. I withdraw my feedback about it on this PR.

**Review by dhh1128 — APPROVED** [review]

File: concepts/0020-message-types/README.md (hunk 2 above)
Comment (swcurran) [reply concepts/0020-message-types/README.md]: Corrected with links to DIF and the DIDComm Messaging spec.

**Comment by llorllale** [comment]
@swcurran there are still quite a few references to `did:sov` in your branch. Are you planning on another followup to address those?

```
~/dev/hyperledger/aries-rfcs$ grep -R did:sov | wc -l
113
~/dev/hyperledger/aries-rfcs$
```

**Review by llorllale — APPROVED** [review]

**Comment by swcurran** [comment]
> @swcurran there are still quite a few references to `did:sov` in your branch. Are you planning on another followup to address those?
> 
> ```
> ~/dev/hyperledger/aries-rfcs$ grep -R did:sov | wc -l
> 113
> ~/dev/hyperledger/aries-rfcs$
> ```

This PR changes references to the literal string `did:sov:BzCbsNYhMrjHiqZDTUASHg;spec`.  The other references are likely examples vs. message type references.  I did a quick scan of this instances and they all (most?) are references to example DIDs, not message type prefixes.  I think all good.

File: concepts/0020-message-types/README.md (hunk 2 above)
Comment (kdenhartog) [reply concepts/0020-message-types/README.md]: Thanks for maintaining control of that and transferring to DIF as well @dhh1128 up until this point.

**Review by kdenhartog — APPROVED** [review]

**Pull request merged**